helm/kubernetes-nginx-ingress-controller-chart/values.yaml

```diff
@@ -16,6 +16,12 @@ configmap:
   server-tokens: "false"
   worker-processes: "4"
 
+  autoscaling:
+    minReplicas: 2
+    maxReplicas: 15
```

Comment:
why did you go so low now? The thing is the default for now will be rolled to all customers. I'm pretty sure a lot of our customers go way higher than 15 (I've seen 64)

Reply:
I want to check on `Asgard` talked to vaclav and he recommended lower. Going to test this a bit, none of these values are set in stone yet.

Reply:
> why did you go so low now? The thing is the default for now will be rolled to all customers. I'm pretty sure a lot of our customers go way higher than 15 (I've seen 64)

I'm genuinely interested in load metrics when someone wants to run even 15 nginx's not to say 64. I know on one customer there can be total of 64 but that's more of a mistake than a good practice. Single nginx can drive tons of load given that HW has enough resources and AFAIK we don't have a single tenant with hundreds of nodes which would justify that high number of instances. Please educate me :slightly_smiling_face:

Reply:
Oh, one more thing in here: Was it number of worker processes for single nginx IC or number of IC PODs? Ping @puja108 

I'm asking this because default config is `num_workers` == `num CPU cores in node` which leads also to often wrong configuration when running on very big instance types (but this can be overridden with IC annotations).

Reply:
IC pods, and partly that is not because of nginx being the bottle neck but the EC2 instance networking (which also depends on the type of machine), I think gitlab had 11, so might be ok, but like Tuomas is saying, it is dependent on the machine it runs on. Let's just play around and we'll see with time. maybe @giantswarm/sig-customer then needs to talk to customers and clarify that they might not need that many